code/modules/spells/spell_types/bloodcrawl.dm

```diff
@@ -23,14 +24,20 @@
 	to_chat(user, "<span class='warning'>There must be a nearby source of blood!</span>")
 
 /obj/effect/proc_holder/spell/bloodcrawl/perform(obj/effect/decal/cleanable/target, recharge = 1, mob/living/user = usr)
-	if(istype(user))
-		if(phased)
-			if(user.phasein(target))
-				phased = FALSE
-		else
-			if(user.phaseout(target))
-				phased = TRUE
-		start_recharge()
+	if(!istype(user))
+		revert_cast()
+		to_chat(user, "<span class='warning'>You are unable to blood crawl!</span>")
 		return
-	revert_cast()
-	to_chat(user, "<span class='warning'>You are unable to blood crawl!</span>")
+
+	if(phased)
+		if(!COOLDOWN_FINISHED(src, resurface_cooldown))
+			to_chat(user, "<span class='warning'>You need to wait until you can resurface!</span>")
+			return
+		if(user.phasein(target))
+			phased = FALSE
+	else
+		if(user.phaseout(target))
+			phased = TRUE
+			COOLDOWN_START(src, resurface_cooldown, 15 SECONDS)
```

Comment:
```suggestion
			COOLDOWN_START(src, resurface_cooldown, 5 SECONDS)
```

Start small, we can increase it more later if it's still necessary, but even a 5 second cooldown will be a major nerf to the current gamer strats with the demons. We don't want crew to have enough time to completely clean the room or else it becomes way too easy to completely shut the demon down.